### ISSUE TITLE
VHACD.GenerateConvexMeshes() never calls DestroyVHACD()

### ISSUE THREAD

**Issue by garettbass** [Unity-Technologies/VHACD-deprecated#5]
The `vhacd` instance created in `VHACD.GenerateConvexMeshes()`: 
```cs
// VHACD.cs:138
var vhacd = CreateVHACD();
```
Should probably be destroyed before returning from that method to avoid a memory leak, e.g.:
```cs
// VHACD.cs:185
DestroyVHACD(vhacd); // proposed change
return convexMesh;
```

**Comment by peifeng-unity** [comment]
Sorry for this late response. Thank you for bringing it up. I have created an internal ticket AIRO-1486 and we will follow up on this issue.

**Comment by at669** [comment]
Hi @garettbass, this has been added and resolved in #7, which is merged into the `dev` branch. I will be closing this issue, but feel free to reopen to continue discussion, or open a new issue if a new topic comes up. Thank you!
